# Практическая работа №2: Обработка выборочных данных. Нахождение точечных оценок параметров распределения

Выполнили студенты гр. 2383 Борисов Иван Вадимович и Сыздыков Нургалым Конакбаевич. Вариант №25

## Цель работы

Получение практических навыков нахождения точечных статистических оценок параметров распределения. 


## Краткое изложение основных теоретических понятий

Статистической оценкой $\Theta^*$ неизвестного параметра теоретического распределения $\Theta$ называется функция от наблюдаемых значений случайной величины:
$$\Theta^* = f(x_1, x_2, \dots, x_n)$$
Статистические оценки, определяемые одним числом, называются точечными.

Для того, чтобы оценки были достаточно надежными, к ним предъявляются требованиия несмещенности, состоятельности и эффективности.

Оценка называется несмещенной, если её математическое ожидание равно оцениваемому параметру $\Theta$ при любом объеме выборки n, т.е.:
$$M(\Theta^*) = \Theta$$

Оценка $\Theta^*$ называется эффективной, если при заданном объеме выборки n она имеет наименьшую возможную дисперсию, т.е. $$D(\Theta^*) \rightarrow \min$$

Оценка $\Theta^*$ называется состоятельной для параметра $\Theta$, если она сходится по вероятности к оцениваемому параметру при неограниченном возрастании объема выборки, т.е.:
$$\lim_{N \rightarrow \infty} P (|\Theta - \Theta^*| \le \varepsilon) = 1$$
где $\varepsilon > 0$ сколь угодно малое положительное число.

Для того, чтобы несмещенная оценка была состоятельной, достаточно чтобы было выполнено условие:
$$\lim_{N \rightarrow \infty} D(\Theta^*) = 0$$

Начальным эмпирическим моментом k-го порядка называется среднее значение k-х степеней элементов вариационного или интервального ряда:
$$\overline{M_k} = \frac{1}{N} \sum n_j x_j^k$$

В частности
$$\overline{x}_в = \overline{M}_1 = \frac{1}{N} \sum n_j x_j$$

Центральным эмпирическим моментом k-го порядка называется среднее значение k-х степеней разностей $(x_j - \overline{x}_в)$ для вариационного или интервального ряда:
$$\overline{m}_k = \frac{1}{N} \sum n_j(x_j - \overline{x}_в)^k$$

В частности
$$D_в = \overline{m_2} = \frac{1}{N} \sum n_j (x_j - \overline{x_в})^2$$

Следует отметить, что указанная статистическая оценка дисперсии является смещенной оценкой. Поэтому следует использовать так называемую исправленную оценку дисперсии:
$$s^2 = \frac{N}{N - 1}D_в$$

Статистические оценки СКО вычисляются как корень квадратный  из соответствующих оценок дисперсии:
$$\sigma_в = \sqrt{D_в} \quad \quad \quad \quad s = \sqrt{s^2}$$

Статистические оценки асимметрии и эксцесса вычисляются по формулам:
$$\overline{A}_s = \frac{\overline{m}_3}{s^3} \quad \quad \quad \quad \overline{E} = \frac{\overline{m_4}}{s^4} - 3$$

Для упрощения вычислений эмпирических моментов вводят в рассмотрение так называемые условные варианты

$$u_j = \frac{x_j - C}{h}$$

где $C$ - условный ноль, значение которого выбирается равным значению варианты интервального ряда, являющейся средней или близкой к средней по значению в этом ряу. В результате все условные варианты оказываются целыми числами. 


Вводятся в рассмотрение условные моменты k-го порядка:

$$\overline{M}_k^* = \frac{1}{N}\sum n_j \left( \frac{x_j - C}{h} \right)^k = \frac{1}{N} \sum n_j u_j^k$$



## Постановка задачи с кратким описанием порядка выполнения работы

Для заданных выборочных данных вычислить с использованием метода моментов и условных вариант точечные статистические оценки математического ожидания, дисперсии, среднеквадратичного отклонения, асимметрии, эксцесса, моды, медианы и коэффициента вариации исследуемой случайной величины. Полученные результаты содержательно проинтерпретировать. 


### Порядок выполнения работы

1. Для середин интервального ряда, полученного в практической работе №1, вычислить условные варианты. Заполнить табл. 1 (в последней строке $\Sigma$ необходимо заполнить суммы столбцов; ячейки отмеченные прочерком заполнять не надо). Провести контроль вычислений.
2. Вычислить условные эмпирические моменты $\overline{M}_{k}^{*}$ через условные варианты. С помощью условных эмпирических моментов вычислить центральные эмпирические моменты $\overline{m}_k$. Полученные результаты занести в табл. 2.
3. Вычислить выборочные среднее $\overline{x}_{в}$ и дисперсию $D_{в}$ с помощью стандартной формулы и с помощью условных вариант. Убедиться, что результаты совпадают. Вычислить выборочное СКО $\sigma_в$.
4. Вычислить исправленную выборочную дисперсию $s^2$ и исправленное СКО $s$. Сравнить данные оценки с смещёнными оценками дисперсии и СКО. Сделать выводы.
5. Найти статистическую оценку коэффициентов асимметрии $\overline{A}_s$ и эксцесса $\overline{E}$. Сделать выводы.
6. Для интервального ряда вычислить моду $M^*_o$, медиану $M^*_e$ и коэффициент вариации $V^*$ заданного распределения. Сделать выводы.

### 1. Для середин интервального ряда, полученного в практической работе №1, вычислить условные варианты. Заполнить табл. 1 (в последней строке $\Sigma$ необходимо заполнить суммы столбцов; ячейки отмеченные прочерком заполнять не надо). Провести контроль вычислений.

In [1]:
import numpy as np
import pandas as pd
from IPython.display import display, Markdown


pd.options.display.float_format = '{:,.4f}'.format

In [2]:
df: pd.DataFrame = pd.read_csv('Syzdykov_Borisov_data_116.csv')

In [3]:
df

,nu,E
0,396,90.1000
1,525,145.3000
2,437,129.4000
3,411,115.2000
4,510,124.1000
...,...,...
111,463,144.9000
112,487,146.0000
113,528,163.4000
114,331,74.1000


In [4]:
def get_interval_stats(series: pd.Series) -> tuple[np.ndarray, np.ndarray, float, float]:
    n: int = len(series)
    k: int = int(np.ceil(1 + 3.31 * np.log10(n)))
    
    x_min: float = series.min()
    x_max: float = series.max()
    h: float = (x_max - x_min) / k
    
    bins: np.ndarray = np.linspace(x_min, x_max, k + 1)
    counts, edges = np.histogram(series, bins=bins)
    midpoints: np.ndarray = (edges[:-1] + edges[1:]) / 2
    
    return midpoints, counts, h, x_min

In [5]:
def create_table_1(midpoints: np.ndarray, counts: np.ndarray, h: float, label: str) -> tuple[pd.DataFrame, float, float]:
    """Формирование расчетной таблицы №1 с проверкой контроля."""
    max_idx: int = np.argmax(counts)
    C: float = midpoints[max_idx]
    
    # Расчет условных вариант
    u: np.ndarray = np.round((midpoints - C) / h).astype(int)
    
    n_u: np.ndarray = counts * u
    n_u2: np.ndarray = counts * (u**2)
    n_u3: np.ndarray = counts * (u**3)
    n_u4: np.ndarray = counts * (u**4)
    n_u_plus_1_4: np.ndarray = counts * ((u + 1)**4)
    
    df_t1: pd.DataFrame = pd.DataFrame({
        r'$i$': range(1, len(midpoints) + 1),
        r'$x_i$': midpoints,
        r'$n_i$': counts,
        r'$u_i$': u,
        r'$n_i \cdot u_i$': n_u,
        r'$n_i \cdot u_i^2$': n_u2,
        r'$n_i \cdot u_i^3$': n_u3,
        r'$n_i \cdot u_i^4$': n_u4,
        r'$n_i \cdot (u_i+1)^4$': n_u_plus_1_4
    })
    
    sums: dict = df_t1.sum()
    sum_row: pd.DataFrame = pd.DataFrame([{
        '$i$': r'$\Sigma$', r'$x_i$': np.nan, r'$n_i$': sums[r'$n_i$'], r'$u_i$': np.nan,
         r'$n_i \cdot u_i$': sums[r'$n_i \cdot u_i$'], r'$n_i \cdot u_i^2$': sums[r'$n_i \cdot u_i^2$'],
         r'$n_i \cdot u_i^3$': sums[r'$n_i \cdot u_i^3$'], r'$n_i \cdot u_i^4$': sums[r'$n_i \cdot u_i^4$'],
         r'$n_i \cdot (u_i+1)^4$': sums[r'$n_i \cdot (u_i+1)^4$']
    }])
    
    lhs: float = sums[r'$n_i \cdot (u_i+1)^4$']
    rhs: float = (sums[r'$n_i \cdot u_i^4$'] + 4*sums[r'$n_i \cdot u_i^3$'] + 6*sums[r'$n_i \cdot u_i^2$'] + 4*sums[r'$n_i \cdot u_i$'] + sums[r'$n_i$'])
    
    print(f"--- Контроль вычислений для {label} ---")
    display(Markdown(rf"$\sum n_i (u_i+1)^4 = {lhs}$"))
    display(Markdown(rf"$\sum n_i \cdot u_i^4 + 4 \sum n_i \cdot u_i^3 + 6 \sum n_i \cdot u_i^2 + 4 \sum n_i \cdot u_i + N = {rhs}$"))
    print(rf"Контроль пройден: {np.isclose(lhs, rhs)}")
    
    return pd.concat([df_t1, sum_row], ignore_index=True), C, h

In [6]:
mid_nu, counts_nu, h_nu, _ = get_interval_stats(df['nu'])
table1_nu, C_nu, h_nu_final = create_table_1(mid_nu, counts_nu, h_nu, "признака nu")
display(Markdown("**Таблица 1 (nu):**"))
display(table1_nu)

--- Контроль вычислений для признака nu ---


$\sum n_i (u_i+1)^4 = 3722.0$

$\sum n_i \cdot u_i^4 + 4 \sum n_i \cdot u_i^3 + 6 \sum n_i \cdot u_i^2 + 4 \sum n_i \cdot u_i + N = 3722.0$

Контроль пройден: True


**Таблица 1 (nu):**

,$i$,$x_i$,$n_i$,$u_i$,$n_i \cdot u_i$,$n_i \cdot u_i^2$,$n_i \cdot u_i^3$,$n_i \cdot u_i^4$,$n_i \cdot (u_i+1)^4$
0,1,338.9375,7.0000,-3.0000,-21.0000,63.0000,-189.0000,567.0000,112.0000
1,2,376.8125,9.0000,-2.0000,-18.0000,36.0000,-72.0000,144.0000,9.0000
2,3,414.6875,22.0000,-1.0000,-22.0000,22.0000,-22.0000,22.0000,0.0000
3,4,452.5625,33.0000,0.0000,0.0000,0.0000,0.0000,0.0000,33.0000
4,5,490.4375,26.0000,1.0000,26.0000,26.0000,26.0000,26.0000,416.0000
5,6,528.3125,14.0000,2.0000,28.0000,56.0000,112.0000,224.0000,"1,134.0000"
6,7,566.1875,3.0000,3.0000,9.0000,27.0000,81.0000,243.0000,768.0000
7,8,604.0625,2.0000,4.0000,8.0000,32.0000,128.0000,512.0000,"1,250.0000"
8,$\Sigma$,NaN,116.0000,NaN,10.0000,262.0000,64.0000,"1,738.0000","3,722.0000"


In [7]:
mid_e, counts_e, h_e, _ = get_interval_stats(df['E'])
table1_e, C_e, h_e_final = create_table_1(mid_e, counts_e, h_e, "признака E")
display(Markdown("**Таблица 1 (E):**"))
display(table1_e)

--- Контроль вычислений для признака E ---


$\sum n_i (u_i+1)^4 = 2955.0$

$\sum n_i \cdot u_i^4 + 4 \sum n_i \cdot u_i^3 + 6 \sum n_i \cdot u_i^2 + 4 \sum n_i \cdot u_i + N = 2955.0$

Контроль пройден: True


**Таблица 1 (E):**

,$i$,$x_i$,$n_i$,$u_i$,$n_i \cdot u_i$,$n_i \cdot u_i^2$,$n_i \cdot u_i^3$,$n_i \cdot u_i^4$,$n_i \cdot (u_i+1)^4$
0,1,78.8875,6.0000,-3.0000,-18.0000,54.0000,-162.0000,486.0000,96.0000
1,2,94.4625,10.0000,-2.0000,-20.0000,40.0000,-80.0000,160.0000,10.0000
2,3,110.0375,15.0000,-1.0000,-15.0000,15.0000,-15.0000,15.0000,0.0000
3,4,125.6125,34.0000,0.0000,0.0000,0.0000,0.0000,0.0000,34.0000
4,5,141.1875,34.0000,1.0000,34.0000,34.0000,34.0000,34.0000,544.0000
5,6,156.7625,14.0000,2.0000,28.0000,56.0000,112.0000,224.0000,"1,134.0000"
6,7,172.3375,2.0000,3.0000,6.0000,18.0000,54.0000,162.0000,512.0000
7,8,187.9125,1.0000,4.0000,4.0000,16.0000,64.0000,256.0000,625.0000
8,$\Sigma$,NaN,116.0000,NaN,19.0000,233.0000,7.0000,"1,337.0000","2,955.0000"


### 2. Вычислить условные эмпирические моменты $\overline{M}_{k}^{*}$ через условные варианты. С помощью условных эмпирических моментов вычислить центральные эмпирические моменты $\overline{m}_k$. Полученные результаты занести в табл. 2.

In [8]:
def calculate_moments(table1: pd.DataFrame, h: float) -> pd.DataFrame:
    """Вычисление условных и центральных моментов."""
    row_sum: pd.Series = table1.iloc[-1]
    n: float = row_sum[r'$n_i$']

    # Условные моменты
    M1_star: float = row_sum[r'$n_i \cdot u_i$'] / n
    M2_star: float = row_sum[r'$n_i \cdot u_i^2$'] / n
    M3_star: float = row_sum[r'$n_i \cdot u_i^3$'] / n
    M4_star: float = row_sum[r'$n_i \cdot u_i^4$'] / n
    
    # Центральные моменты (через условные)
    m1: float = 0.0
    m2: float = (M2_star - M1_star**2) * (h**2)
    m3: float = (M3_star - 3*M1_star*M2_star + 2*M1_star**3) * (h**3)
    m4: float = (M4_star - 4*M1_star*M3_star + 6*M1_star**2*M2_star - 3*M1_star**4) * (h**4)
    
    return pd.DataFrame({
        r'$k$': [1, 2, 3, 4],
        r'$\overline{M}^*_k$': [M1_star, M2_star, M3_star, M4_star],
        r'$\overline{m}_k$': [m1, m2, m3, m4]
    })


In [9]:
moments_nu: pd.DataFrame = calculate_moments(table1_nu, h_nu_final)
display(Markdown("**Таблица 2 (nu):**"))
display(moments_nu)

**Таблица 2 (nu):**

,$k$,$\overline{M}^*_k$,$\overline{m}_k$
0,1,0.0862,0.0000
1,2,2.2586,"3,229.3659"
2,3,0.5517,"-1,690.8524"
3,4,14.9828,"30,647,450.2147"


In [10]:
moments_e: pd.DataFrame = calculate_moments(table1_e, h_e_final)
display(Markdown("**Таблица 2 (E):**"))
display(moments_e)

**Таблица 2 (E):**

,$k$,$\overline{M}^*_k$,$\overline{m}_k$
0,1,0.1638,0.0000
1,2,2.0086,480.7445
2,3,0.0603,"-3,467.8573"
3,4,11.5259,"694,816.1355"


### 3. Вычислить выборочные среднее $\overline{x}_{в}$ и дисперсию $D_{в}$ с помощью стандартной формулы и с помощью условных вариант. Убедиться, что результаты совпадают. Вычислить выборочное СКО $\sigma_в$.

In [11]:
def compute_main_stats(midpoints: np.ndarray, counts: np.ndarray, moments: pd.DataFrame, C: float, h: float, label: str) -> tuple[float, float, float]:
    """Сравнение методов расчета внутри интервального ряда."""
    # 1. Метод условных вариант
    m1_star: float = moments.loc[0, r'$\overline{M}^*_k$']
    x_avg_cond: float = m1_star * h + C
    d_v_cond: float = moments.loc[1, r'$\overline{m}_k$']
    
    # 2. Стандартная формула для интервального ряда
    x_avg_int: float = np.average(midpoints, weights=counts)
    d_v_int: float = np.average((midpoints - x_avg_int)**2, weights=counts)
    
    # 3. Выборочное СКО
    sigma_v: float = np.sqrt(d_v_cond)
    
    display(Markdown(f"#### Результаты для {label}:"))
    
    comparison_df: pd.DataFrame = pd.DataFrame({
        "Показатель": ["Среднее значение", "Выборочная дисперсия"],
        "Стандартная формула": [x_avg_int, d_v_int],
        "Условные варианты": [x_avg_cond, d_v_cond],
        "Разность": [abs(x_avg_int - x_avg_cond), abs(d_v_int - d_v_cond)]
    })
    display(comparison_df)
    
    display(Markdown(rf"Выборочное СКО ($\sigma_в$): **{sigma_v:.4f}**"))
    return x_avg_cond, d_v_cond, sigma_v


In [12]:
x_nu, d_nu, sigma_nu = compute_main_stats(mid_nu, counts_nu, moments_nu, C_nu, h_nu_final, "nu")

#### Результаты для nu:

,Показатель,Стандартная формула,Условные варианты,Разность
0,Среднее значение,455.8276,455.8276,0.0000
1,Выборочная дисперсия,"3,229.3659","3,229.3659",0.0000


Выборочное СКО ($\sigma_в$): **56.8275**

In [13]:
x_e, d_e, sigma_e = compute_main_stats(mid_e, counts_e, moments_e, C_e, h_e_final, "E")

#### Результаты для E:

,Показатель,Стандартная формула,Условные варианты,Разность
0,Среднее значение,128.1636,128.1636,0.0000
1,Выборочная дисперсия,480.7445,480.7445,0.0000


Выборочное СКО ($\sigma_в$): **21.9259**

### 4. Вычислить исправленную выборочную дисперсию $s^2$ и исправленное СКО $s$. Сравнить данные оценки с смещёнными оценками дисперсии и СКО. Сделать выводы.

In [14]:
def calculate_corrected_estimates(d_v: float, n: int, label: str) -> tuple[float, float]:
    s2: float = (n / (n - 1)) * d_v
    s: float = np.sqrt(s2)
    
    display(Markdown(f"#### Исправленные оценки для {label}:"))
    display(Markdown(rf"- Исправленная дисперсия ($s^2$): **{s2:.4f}**"))
    display(Markdown(rf"- Исправленное СКО ($s$): **{s:.4f}**"))
    
    return s2, s


n_val: int = len(df)
s2_nu, s_nu = calculate_corrected_estimates(d_nu, n_val, "nu")
s2_e, s_e = calculate_corrected_estimates(d_e, n_val, "E")

#### Исправленные оценки для nu:

- Исправленная дисперсия ($s^2$): **3257.4473**

- Исправленное СКО ($s$): **57.0741**

#### Исправленные оценки для E:

- Исправленная дисперсия ($s^2$): **484.9249**

- Исправленное СКО ($s$): **22.0210**

#### Выводы по исправленным оценкам:
1. **Сравнение оценок:** Исправленная дисперсия $s^2$ и исправленное СКО $s$ оказались больше соответствующих смещенных оценок ($D_в$ и $\sigma_в$). Это математически обусловлено использованием поправки Бесселя $\frac{n}{n-1}$.
2. **Анализ смещения:** Выборочная дисперсия является смещенной оценкой, так как она в среднем занижает значение дисперсии. Разница между оценками для обоих признаков составляет менее **1%**, что объясняется достаточно большим объемом выборки ($n=116$). 
3. **Практическая значимость:** Несмотря на малую разницу, для построения дальнейших интервальных оценок и проверки гипотез необходимо использовать именно исправленные значения, так как они обеспечивают отсутствие систематической ошибки.

### 5. Найти статистическую оценку коэффициентов асимметрии $\overline{A}_s$ и эксцесса $\overline{E}$. Сделать выводы.

In [15]:
def analyze_distribution_shape(moments: pd.DataFrame, sigma_v: float, label: str) -> None:
    """Расчет асимметрии и эксцесса."""
    m3: float = moments.loc[2, r'$\overline{m}_k$']
    m4: float = moments.loc[3, r'$\overline{m}_k$']
    
    # Асимметрия и эксцесс (используем выборочное СКО)
    as_val: float = m3 / (sigma_v**3)
    e_val: float = (m4 / (sigma_v**4)) - 3
    
    display(Markdown(f"#### Характеристики формы для {label}:"))
    display(Markdown(rf"- Коэффициент асимметрии ($\overline{{A}}_s$): **{as_val:.4f}**"))
    display(Markdown(rf"- Коэффициент эксцесса ($\overline{{E}}$): **{e_val:.4f}**"))


analyze_distribution_shape(moments_nu, sigma_nu, "nu")
analyze_distribution_shape(moments_e, sigma_e, "E")

#### Характеристики формы для nu:

- Коэффициент асимметрии ($\overline{A}_s$): **-0.0092**

- Коэффициент эксцесса ($\overline{E}$): **-0.0613**

#### Характеристики формы для E:

- Коэффициент асимметрии ($\overline{A}_s$): **-0.3290**

- Коэффициент эксцесса ($\overline{E}$): **0.0064**

#### Выводы по характеристикам формы распределения:

**Для признака `nu`:**
* **Асимметрия ($\overline{A}_s \approx -0.0092$):** Коэффициент крайне близок к нулю. Это свидетельствует о том, что распределение практически симметрично. Наблюдается лишь ничтожно малая левосторонняя асимметрия (незначительный «хвост» в области малых значений).
* **Эксцесс ($\overline{E} \approx -0.0613$):** Значение близко к нулю и отрицательно. Распределение является чуть более пологим, чем стандартное нормальное распределение.

**Для признака `E`:**
* **Асимметрия ($\overline{A}_s \approx -0.3290$):** Наблюдается умеренная левосторонняя асимметрия. Это означает, что большинство значений сосредоточено справа от среднего, а в левой части («хвосте») находятся редкие, более низкие значения.
* **Эксцесс ($\overline{E} \approx 0.0064$):** Значение практически равно нулю, что говорит о том, что по «островершинности» данное распределение почти идентично теоретическому нормальному распределению.

### 6. Для интервального ряда вычислить моду $M^*_o$, медиану $M^*_e$ и коэффициент вариации $V^*$ заданного распределения. Сделать выводы.

In [16]:
def compute_robust_and_variation(
    midpoints: np.ndarray, 
    counts: np.ndarray, 
    h: float, 
    x_avg: float, 
    sigma_v: float, 
    label: str
) -> None:
    """Расчет моды, медианы и коэффициента вариации."""
    # 1. Коэффициент вариации
    v_star: float = (sigma_v / x_avg) * 100
    
    # 2. Мода (Mo)
    max_idx: int = np.argmax(counts)
    # Нижняя граница модального интервала
    l_mo: float = midpoints[max_idx] - h/2
    n_mo: int = counts[max_idx]
    n_mo_prev: int = counts[max_idx - 1] if max_idx > 0 else 0
    n_mo_next: int = counts[max_idx + 1] if max_idx < len(counts) - 1 else 0
    
    mo_star: float = l_mo + h * (n_mo - n_mo_prev) / ((n_mo - n_mo_prev) + (n_mo - n_mo_next))
    
    # 3. Медиана (Me)
    half_n: float = np.sum(counts) / 2
    cum_counts: np.ndarray = np.cumsum(counts)
    
    me_idx: int = np.where(cum_counts >= half_n)[0][0]
    l_me: float = midpoints[me_idx] - h/2
    s_me_prev: float = cum_counts[me_idx - 1] if me_idx > 0 else 0
    n_me: int = counts[me_idx]
    
    me_star: float = l_me + h * (half_n - s_me_prev) / n_me
    
    display(Markdown(f"#### Итоговые показатели для {label}:"))
    display(Markdown(rf"- Мода ($M_o^*$): **{mo_star:.4f}**"))
    display(Markdown(rf"- Медиана ($M_e^*$): **{me_star:.4f}**"))
    display(Markdown(rf"- Коэффициент вариации ($V^*$): **{v_star:.2f}%**"))


compute_robust_and_variation(mid_nu, counts_nu, h_nu_final, x_nu, sigma_nu, "nu")
compute_robust_and_variation(mid_e, counts_e, h_e_final, x_e, sigma_e, "E")

#### Итоговые показатели для nu:

- Мода ($M_o^*$): **456.7708**

- Медиана ($M_e^*$): **456.5795**

- Коэффициент вариации ($V^*$): **12.47%**

#### Итоговые показатели для E:

- Мода ($M_o^*$): **133.4000**

- Медиана ($M_e^*$): **130.1934**

- Коэффициент вариации ($V^*$): **17.11%**

#### Выводы по характеристикам распределения
1. **Коэффициент вариации ($V^*$):** Для обоих исследуемых признаков коэффициент вариации не превышает порог в **33%** (12.47% и 17.11% соответственно). Это означает что исследуемая совокупность данных однородна, а средние значения типичны для данной выборки.
2. **Мода и медиана:**
   * Для признака **nu** мода, медиана и выборочное среднее практически равны между собой ($\approx 456$), что подтверждает симметричность распределения.
   * Для признака **E** наблюдается соотношение $M_o^* > M_e^* > \overline{x}_в$ ($133.4 > 130.2 > 128.2$). Такое неравенство статистически подтверждает наличие левосторонней асимметрии: большинство значений в выборке сосредоточено в правой части интервала (выше среднего).

## Вывод

В ходе выполнения практической работы были освоены методы нахождения точечных статистических оценок параметров распределения для двух признаков экспериментальных данных. 

1. С помощью метода условных вариант и ложного нуля были успешно рассчитаны эмпирические моменты до 4-го порядка включительно. Контроль вычислений подтвердил правильность заполнения расчетных таблиц.
2. Подтверждено совпадение результатов, полученных классическим методом и методом условных вариант, что доказывает вычислительную эффективность последнего при работе с интервальными рядами.
3. Проведен анализ однородности: коэффициент вариации для обоих признаков не превысил порог в 33%, что позволяет классифицировать выборку как однородную.
4. Установлено, что распределение признака `nu` практически симметрично и близко к нормальному, в то время как признак `E` обладает выраженным левосторонним скосом. 
5. Все полученные оценки (среднее, дисперсия, СКО) являются точечными и служат фундаментом для дальнейшего статистического анализа, включая проверку гипотез о законе распределения и построение доверительных интервалов.